shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/dml/impl/DeleteStatementAssert.java

```diff
@@ -49,8 +51,10 @@
     public static void assertIs(final SQLCaseAssertContext assertContext, final DeleteStatement actual, final DeleteStatementTestCase expected) {
         assertTable(assertContext, actual, expected);
         assertWhereClause(assertContext, actual, expected);
-        assertOrderByClause(assertContext, actual, expected);
-        assertLimitClause(assertContext, actual, expected);
+        if (actual instanceof MySQLStatement) {
+            assertOrderByClause(assertContext, (MySQLDeleteStatement) actual, expected);
+            assertLimitClause(assertContext, (MySQLDeleteStatement) actual, expected);    
+        }
```

Comment:
How about to extract A function named `assertXXXXX`

Reply:
It is an optional item. Just from my glance, my feeling is that it is better to gather this judgment and handling detail into a function.

Reply:
> It is an optional item. Just from my glance, my feeling is that it is better to gather this judgment and handling detail into a function.

It is a good idea. 👍